flask_minify/main.py

```diff
@@ -19,7 +19,7 @@ class Minify(object):
     'Extension to minify flask response for html, javascript, css and less.'
 
     def __init__(
-        self, app=None, html=True, js=True, cssless=True,
+        self, app=None, html=True, js=True, script_types=[], cssless=True,
         fail_safe=True, bypass=[], bypass_caching=[], caching_limit=2,
         passive=False, static=True
```

Comment:
Let's move `script_types` to the end of keyword arguments, to avoid introducing a breaking change for anyone's using arguments only:

```suggestion
        self, app=None, html=True, js=True, cssless=True,
        fail_safe=True, bypass=[], bypass_caching=[], caching_limit=2,
        passive=False, static=True, script_types=[]
```